docs/gen-ai/gen-ai-system-instructions.json

```diff
@@ -30,6 +30,19 @@
                     "description": "Text content sent to or received from the model.",
                     "title": "Content",
                     "type": "string"
+                },
+                "name": {
```

Comment:
It's currently on the `TextPart`, but I assume you want it to be on the `ChatMessage` (all parts of the same message belong to the same participant)